runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -120,6 +130,8 @@
 	private LootTrackerPanel panel;
 	private NavigationButton navButton;
 	private String eventType;
+	// Brimstone chest loot message
```

Comment:
Name the variable better so this comment isn't needed. `brimstoneChestMessage` maybe?